atomic_reactor/plugins/cachi2_postprocess.py

```diff
@@ -110,13 +111,38 @@ def run(self) -> Optional[List[Dict[str, Any]]]:
             return None
 
         processed_remote_sources = self.postprocess_remote_sources()
+        self.postprocess_git_submodules_global_sbom()
         self.inject_remote_sources(processed_remote_sources)
 
         return [
             self.remote_source_to_output(remote_source)
             for remote_source in processed_remote_sources
         ]
 
+    def postprocess_git_submodules_global_sbom(self):
+        """atomic-reactor is responsbile for handling git-submodules. Global SBOM must be updated"""
+        all_sboms = []
+        for remote_source in self.init_plugin_data:
+            git_submodules = remote_source.get('git_submodules')
+            if not git_submodules:
+                continue
+
+            all_sboms.extend(git_submodules['sbom'])
+
+        if not all_sboms:
+            return
+
+        # TODO: deduplicate sboms?
```

Comment:
I'm not sure if there should be a deduplication for global sbom in case of that the same submodule is used within multiple remote sources.

Maybe play safe and don't deduplicate, WDYT?

Reply:
Cachito did use to de-duplicate https://github.com/containerbuildsystem/cachito/blob/2e4795a7687b70a468e4f686ef05988994af1a7b/cachito/web/api_v1.py#L931-L939

But CycloneDX doesn't technically require uniqueness if you're not using [bom-ref](https://cyclonedx.org/docs/1.5/json/#components_items_bom-ref) attributes (which we're not), so it's not really necessary